README.md

```diff
@@ -1,71 +1,152 @@
-    - Unique service selector
-  - [Gatekeeper Policy Manager](katalog/gatekeeper/gpm): Gatekeeper Policy Manager, a simple to use web-ui for
-  Gatekeeper. Version: **v0.5.1**
+If you are new to KFD please refer to the [official documentation][kfd-docs] on how to get started with KFD.
 
-You can click on each package to see its documentation.
+## Overview
 
-## Requirements
+The Kubernetes API server provides a mechanism to review every request that is made, being objects creation, modification or deletion. To use this mechanism the API server allow us to create a [Validating Admission Webhook][kubernetes-vaw-docs] that, as the name says, will validate every requests and let the API server know if the request is allowed or not based on some logic (policy).
 
-All packages in this repository have the following dependencies, for package
-specific dependencies please visit the single package's documentation:
+**Kubernetes Fury OPA** module is based on [OPA Gatekeeper][gatekeeper-page], a popular open-source Kubernetes-native policy engine with [OPA](https://www.openpolicyagent.org/) as its core that runs as a Validating Admission Webhook. It allows writing custom constraints (policies) in `rego` (a tailor-made language) as Kubernetes objects and enforce at runtime.
 
-- [Kubernetes](https://kubernetes.io) >= `v1.20.0`
-- [Furyctl](https://github.com/sighupio/furyctl) package manager to download
-    Fury packages >= [`v0.2.2`](https://github.com/sighupio/furyctl/releases/tag/v0.2.2)
-- [Kustomize](https://github.com/kubernetes-sigs/kustomize) = `v3.3.0`
-- [prometheus-operator](https://github.com/sighupio/fury-kubernetes-monitoring/tree/master/katalog/prometheus-operator)
-from the [Fury Monitoring Module](https://github.com/sighupio/fury-kubernetes-monitoring) is required by
-[Service monitor](./katalog/gatekeeper/core/service-monitor.yml) to export metrics to Prometheus.
+[SIGHUP][sighup-page] provides a set of base constraints that could be used both as a starting point to apply constraints to your current workloads or to give you an idea on how to implement new rules matching your requirements.
 
-> You can comment out the service monitor in the [kustomization.yaml](./katalog/gatekeeper/core/kustomization.yaml)
-file if you don't want to install the monitoring module.
+## Packages
+
+Fury Kubernetes OPA provides the following packages:
+
+| Package                                             | Version  | Description                                                                      |
+|-----------------------------------------------------|----------|----------------------------------------------------------------------------------|
+| [Gatekeeper Core](katalog/gatekeeper/core)          | `v3.7.0` | Gatekeeper deployment, ready to apply rules.                                     |
+| [Gatekeeper Rules](katalog/gatekeeper/rules)        | `N.A.`   | A set of custom rules to get started. See the package documentation for details. |
```

Comment:
I will remove "See the package documentation for details."

It is said after the table to click for more details